course/src/main/java/org/openedx/course/presentation/container/CourseContainerFragment.kt

```diff
@@ -155,6 +156,30 @@ class CourseContainerFragment : Fragment(R.layout.fragment_course_container) {
         viewModel.updateData(withSwipeRefresh)
     }
 
+    fun showDatesUpdateSnackbar(isSuccess: Boolean) {
+        val message = if (isSuccess) {
+            getString(coreR.string.core_dates_shift_dates_successfully_msg)
+        } else {
+            getString(coreR.string.core_dates_shift_dates_unsuccessful_msg)
+        }
+        val duration = if (isSuccess) {
+            Snackbar.LENGTH_INDEFINITE
+        } else {
+            Snackbar.LENGTH_LONG
+        }
+        snackBar = Snackbar.make(binding.root, message, duration).also {
+            if (isSuccess) {
+                it.setAction(coreR.string.core_dates_view_all_dates) {
+                    binding.viewPager.setCurrentItem(3, true)
```

Comment:
It's not a good practice to use hardcoded values for the page number, please try to use enum for this